data-generation/src/test/scala/SnapshotDeltaSpec.scala

```diff
@@ -183,4 +183,32 @@ class SnapshotDeltaSpec extends AnyFlatSpec with SparkTestWrapper {
     assert(appliedG.collect().find(x => x._1 == 3).head._2 == create.head.attributes)
 
   }
+
+  "activatedEntities" should "retrieve the activated entities in an interval" in {
+    implicit val sparkContext: SparkContext = spark.sparkContext // Needed for implicit conversion of Seq -> RDD
+
+    val vertexLogs1 = LogFactory(startTime = 2, endTime = 8).buildSingleSequence(VERTEX(1))
+    val vertexLogs2 = LogFactory(startTime = 5, endTime = 10).buildSingleSequence(VERTEX(2))
+    val edgeLogs = LogFactory(startTime = 4, endTime = 7).buildSingleSequence(EDGE(1, 1, 2))
+    val logs = (vertexLogs1 ++ vertexLogs2 ++ edgeLogs).sortBy(_.timestamp)
+    val g = SnapshotDeltaObject.create(logs, SnapshotIntervalType.Time(3))
+    val intervalWithVertex1 = g.activatedEntities(Interval(0, 3))
+    val intervalWithVertex2 = g.activatedEntities(Interval(5, 7))
```

Comment:
```suggestion
    val intervalWithVertex2 = g.activatedEntities(Interval(9, 10))
```
For å ikke aktivere edgen